### DISCUSSION TITLE
Transmog Across Equipable Armor Types

### DISCUSSION THREAD

**Discussion by Ithiloneth** [chromiecraft/chromiecraft#3299] [Suggestions]
**Suggestion:**
Enable transmog across types, so long as the type is equipable to your class.

**The Case For:**
Transmog currently only works within armor types, meaning a mail item cannot be transmoged to have the appearance of a cloth or leather item.

Speaking only for myself, I wear a mix of cloth/leather/mail on my restoration shaman, and so feel this system is not optimal for me to able to have a full outfit that I use consistently.
This is disappointing, and I can't help but feel that there is no good reason for the system to have this restriction as many characters end up wearing armor from all equipable types and thus have mixed appearances anyway.

**The Case Against:**
A case can be made that trolls will use this to ruin the game by changing their warrior to appear as a mage. This will likely happen, but in my experience from other servers it is exceedingly rare. My personal hypothesis is simply that those who have the mindset that such a thing would be fun to troll others, also tend to have the mindset that their big ass sword is way cool, and they'd rather die than be separated from it (I can relate to that).

**A More Ambitious Route:**
An option here could be to enable transmog in several tiers (I think this is feasible):

1. Off (no transmog)
2. Blizzlike (same restrictions as Cataclysm, which is what we have now afaik)
3. Equipable (any item you can equip can be transmoged with any other item, excepting weapons where melee, ranged, and wands are separate types)
4. Free For All (and may the trolls rejoice!)

Setting the level to 3 would display tmog on characters who have their setting at 1, 2, or 3.
Setting the level to 2 would display tmog on characters who have their setting at 1 or 2.

**Comment by Ithiloneth** [comment]
Reserved!

_If you are upvoting; do that on the main post please; not this reserved one.
Thanks! 🙇_

**Comment by Daniel7Ionescu** [comment]
This would open some cool combos and more stuff to farm, no more disenchanting MC loot, Looking forward to see what people can come up with.

**Reply by Ithiloneth** [reply]
Thanks for the input b3zna!
Think you upvoted my reserved post instead of the topic post - if you have time please correct that :]

**Comment by Nyeriah** [comment]
I believe this is something that needs to be addressed with care. I'm certain it would be rather anticlimatic to see warlocks and priests running around in plate gear. The transmog feature seems to be enjoyed by the vast majority of players, it would be a shame to ruin it.

**Reply by Daniel7Ionescu** [reply]
You would still need to be able to equip the item, so no clothies in plate.  That being said, there could be some cool battle-mage Tmogs.
Regarding the weapons, it would be cool to relax the restrictions on those, 1h sword to any other 1hander and the same for 2handers. No point in ruining a set when you upgrade your weapon.

**Comment by CptGermania** [comment]
As an Hpala, i am 100% of the opinion that this should be implemented. I feel like that the downsides you describe (like warries dressing up as mages) are actually cool new sets to try out and are not taking anything away from anyone. Except in pvp but then turn that shit off and you are good.

**Reply by Ithiloneth** [reply]
This is a very wholesome opinion I think - since you can with just a click of a button turn it on and off, it can always be disabled if there is a problem with the game experience due to outlines becoming unclear.

I do see how it could be an issue for those who want transmog to be turned on, without having to worry about this, but if people want to troll, there are already in-type armor pieces out there to let you hide your class if you want to.

Must be nice for all the cloth only classes - all their armor transmog would just work, no matter what new gear they find.
_(Not jealous at all!)_

**Comment by Cooties-cc** [comment]
I'm semi surprised I haven't seen anyone make the argument that the server is technically an RP server. Of course no one actually follows the blizz rulesets for RP servers, but it's still a big leverage point that could be made. Since Tmog isn't native to Wotlk client, the rules aren't hard set either. However the systems could be worked on a little bit. 

I've seen some other systems that do include Tmog are done so by tokens. Of which you use 1 token per mog, and they are divided into how you can obtain them. The main way was to buy them from the store. This drove up donations to the server, while still getting something out of it. The other 2 were split between PvE/PvP, and both are based around fragments. PvE you could get fragments from RDF bags, killing the last boss of a dungeon (non-rdf/specific que), or by killing raid bosses. You could also include them to drop from chests, but this is semi-abusable. PvP you would get fragments for winning games/KBs. Once you get a certain amount of fragments, it could be crafted into a token, and thus used on a mog. The addition of these fragment systems increased PvP and PvE activity, without hurting the econ or unfairly rewarding certain players. Currently it only costs gold, which can be obtained by anti-social means, and doesn't promote a natural increase to social participation. If there was consideration of this system, but heavy pushback to no-gold, then you could maybe implement a token seller NPC. You would limit daily token purchase to 1, and heavily increase the gold cost. 

I also like the idea of full range Tmog, with just the exception that they would have to match the slot type (1h -> 1h, not 1h -> 2h, etc). You can simplify the tier command systems into just : off, on, RP. RP being the full range of Tmog which would allow greys+ to be moggable. Gear with class restrictions should still apply as they are, in that if you get them on an alt, they can be used acct wide. I don't fully agree that if you enter a BG, the mogs should be removed. Leveling a character, you all look like goofballs in mixed gear, and did just fine playing against each other. At level 60 I see no difference. There are also an untold amount of addons to assist in identifying an enemy before they even get near you. 

I for one would love to see a player charging at me in a santa outfit, with bunny ears, holding a shovel.

**Reply by Ithiloneth** [reply]
Thanks for the feedback - personally I am always hesitant when microtransactions are on the table, and part of what makes CC such a great serer for me is that they have de-emphasized monetization.

It is an interesting option, of course, and if the server _needs_ the extra income I'm not against it, so long as rates for earning tokens in game are reasonable (which tbh nullifies the point in monetizing them, but I digress).

**Comment by PietroColoretti01** [comment]
About this, i think that transmog should allow to consider 1H/2H Maces, Swords and Axes of the same type. We are always talking about weapons, so i don't really see the reason why not to let people transmog an axe into a sword for example.

**Comment by BeefSteaker** [comment]
Should be available to weapons at least. maybe keep to 1H/2H to same type and proficent weapon classes only to limit weirdness of spellcaster with big swords.

**Comment by Frodhi** [comment]
I would say that an an option to "upgrade" the armor type should be available. And not the other way around.
That way, we wouldn't see any warriors in robes, cause lets face it, some find an appeal in "ugly" outfits. And also save the "embarassement" of having to actively show that you wear armor "below" your designated armor type simply because said item is BiS. e.g Truestrike shoulders.
It should be note though that this would have a reduced effect after the implementation of TBC brackets, as the itemisation there is more favourable towards off-specs.
